### PR TITLE
update PBI metrics definition

### PR DESCRIPTION
**Pull request by sarahwxy**
- corrected ODB metrics 
- added the table for tenant office licenses

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/admin/usage-analytics/usage-analytics-data-model.md [hunk 1]
```diff
@@ -104,8 +104,8 @@ This table contains data about each user who had an activity in any of the servi
 |ODB_FileSynched  <br/> |Number of files this user synchronized on any OneDrive for Business.  <br/> |
 |ODB_FileSharedInternally  <br/> |Number of files this user shared internally from any OneDrive for Business, or with users within groups (that might include external users).  <br/> |
 |ODB_FileSharedExternally  <br/> |Number of files this user shared externally from any OneDrive for Business.  <br/> |
-|ODB_AccessByOwner  <br/> |Number of files the user interacted with that reside on their own OneDrive for Business.  <br/> |
-|ODB_AccessOthers  <br/> |Number of files this user interacted with which reside on another user's OneDrive for Business.  <br/> |
+|ODB_AccessedByOwner  <br/> |Number of sites the user interacted with that reside on their own OneDrive for Business.  <br/> |
+|ODB_AccessedByOthers  <br/> |Number of sites this user interacted with which reside on another user's OneDrive for Business.  <br/> |
 |SPO_GroupFileViewedModified  <br/> |Number of files with this user interacted on any group site.  <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/usage-analytics/usage-analytics-data-model.md]:
```suggestion
|SPO_GroupFileViewedModified  <br/> |Number of files with this user interacted with on any group site.  <br/> |
```

File: microsoft-365/admin/usage-analytics/usage-analytics-data-model.md [hunk 2]
```diff
@@ -273,7 +273,17 @@ This table provides data about how Microsoft 365 Groups is used across the organ
...
-   
+
+### Data table - Tenant Office Licenses
+
+This table provides month-over-month summary data about the license assignment for users. 
+  
+|**Column name**|**Column description**|
+|:-----|:-----|
+|LicenseName  <br/> |Name of the license.  <br/> |
+|AssignedCount  <br/> |Number of assigned license.  <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/usage-analytics/usage-analytics-data-model.md]:
```suggestion
|AssignedCount  <br/> |Number of assigned licenses.  <br/> |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @sarahwxy, kindly commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@sarahwxy Please implement the suggested changes. Thanks

File: microsoft-365/admin/usage-analytics/usage-analytics-data-model.md [hunk 3]
```diff
@@ -104,9 +104,9 @@ This table contains data about each user who had an activity in any of the servi
 |ODB_FileSynched  <br/> |Number of files this user synchronized on any OneDrive for Business.  <br/> |
 |ODB_FileSharedInternally  <br/> |Number of files this user shared internally from any OneDrive for Business, or with users within groups (that might include external users).  <br/> |
 |ODB_FileSharedExternally  <br/> |Number of files this user shared externally from any OneDrive for Business.  <br/> |
-|ODB_AccessByOwner  <br/> |Number of files the user interacted with that reside on their own OneDrive for Business.  <br/> |
-|ODB_AccessOthers  <br/> |Number of files this user interacted with which reside on another user's OneDrive for Business.  <br/> |
-|SPO_GroupFileViewedModified  <br/> |Number of files with this user interacted on any group site.  <br/> |
+|ODB_AccessedByOwner  <br/> |Number of sites the user interacted with that reside on their own OneDrive for Business.  <br/> |
+|ODB_AccessedByOthers  <br/> |Number of sites this user interacted with which reside on another user's OneDrive for Business.  <br/> |
+|SPO_GroupFileViewedModified  <br/> |Number of files with this user interacted with on any group site.  <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/usage-analytics/usage-analytics-data-model.md]:
```suggestion
|SPO_GroupFileViewedModified  <br/> |Number of files this user interacted with on any group site.  <br/> |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@sarahwxy - apologies, my bad. Please commit the additional edit. Many thanks.

**Comment by sarahwxy** [comment]
> @sarahwxy - apologies, my bad. Please commit the additional edit. Many thanks.

**Comment by sarahwxy** [comment]
no worries, committed!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@efrene Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@efrene Gentle reminder

**Pull request merged**